docs/grammar.txt

```diff
@@ -32,6 +32,9 @@ EventDefinition = 'event' Identifier EventParameterList 'anonymous'? ';'
 EnumValue = Identifier
 EnumDefinition = 'enum' Identifier '{' EnumValue? (',' EnumValue)* '}'
 
+ModifierArea = 'using modifier' ModifierInvocation (',' ModifierInvocation)*
+               '{' (ModifierArea | FunctionDefinition) '}'
```

Comment:
Is this correct? Can't there be multiple function definitions and modifier areas in a modifier area? Actually there should also be test cases for that (at the moment I only see test cases with a single function definition in a modifier area).

Reply:
Yes. I think I just forgot the asterisk.